mh/pat_item.py

```diff
@@ -307,6 +354,12 @@ def __new__(cls):
         return Custom.__new__(cls, b"\xff", b"\xff")
 
 
+def pack_bytes(*args):
+    """Pack bytes list."""
+    count = len(args)
+    return struct.pack(">B" + count * "B", count, *args)
```

Comment:
I'm not very fond of that `count * "B"` since the `struct.pack` would have to parse that format.
Would something like this also work?
```python
def pack_bytes(*args):
    count = len(args)
    assert count < 0xff
    buff = bytearray(count + 1)
    buff[0] = count
    for i in range(count):
        buff[i + 1] = args[i] & 0xff
    return buff
```

Reply:
I figured out a simpler approach using a bytearray which should be more straight forward:
```
struct.pack(">B", len(args)) + bytearray(args)
```